packages/admin/dashboard/src/routes/orders/order-request-transfer/components/create-order-transfer-form/create-order-transfer-form.tsx

```diff
@@ -0,0 +1,150 @@
+import * as zod from "zod"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { HttpTypes } from "@medusajs/types"
+import { Button, Input, toast } from "@medusajs/ui"
+import { useForm } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+
+import { Form } from "../../../../../components/common/form"
+import { RouteDrawer, useRouteModal } from "../../../../../components/modals"
+import { KeyboundForm } from "../../../../../components/utilities/keybound-form"
+import { useComboboxData } from "../../../../../hooks/use-combobox-data"
+import { Combobox } from "../../../../../components/inputs/combobox"
+import { useRequestTransferOrder } from "../../../../../hooks/api"
+import { sdk } from "../../../../../lib/client"
+import { TransferHeader } from "./transfer-header"
+
+type CreateOrderTransferFormProps = {
+  order: HttpTypes.AdminOrder
+}
+
+const CreateOrderTransferSchema = zod.object({
+  customer_id: zod.string().min(1),
+  currentCustomerDetails: zod.string().min(1),
```

Comment:
though(non-blocking): the difference in casing here is a bit fun, but I guess it's because its used for both the form and the payload

Reply:
good notice, fixed